### PR TITLE
OSDOCS-9825: adds 4.16.0 async relnote MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.16

Issue:
[OSDOCS-9825](https://issues.redhat.com/browse/OSDOCS-9825)

Link to docs preview:
[Bug fixes](https://77258--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes.html#microshift-4-16-bug-fixes)
[4.16.0 async relnote](https://77258--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-16-release-notes.html#microshift-4-16-0-dp)

SME review:
- [x] SMEs have approved this change.

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Review by pacevedom — APPROVED** [review]
/lgtm

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 1]
```diff
@@ -92,8 +92,12 @@ See xref:../microshift_running_apps/microshift-gitops.adoc#microshift-gitops[Aut
...
 With this release, you can get the ID of a {microshift-short} cluster. When opening a support case, you can provide the cluster ID to Red{nbsp}Hat Support to help in identifying issues with your cluster. See xref:../microshift_support/microshift-getting-cluster-id.adoc#microshift-getting-cluster-id[Getting your cluster ID] for more information.
 
-//[id="microshift-4-16-security"]
-//=== Security and compliance
+[id="microshift-4-16-security"]
+=== Security and compliance
+
+[id="microshift-4-16-ssl-medium-cipher-suites"]
+==== SSL Medium Strength Cipher Suites now supported
+During an SSL handshake between a client and a server, the cipher to use is negotiated between them. With this release, SSL Medium Strength Cipher Suites are now supported for the kube-controller-manager daemon, kube-scheduler control-plane process, and kubelet "node agent." This enhancement to the internal communication between kubernetes components improves control plane communications security.
```
Comment (pacevedom) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:100]: Do we want to reference the issue? https://issues.redhat.com/browse/OCPBUGS-29037

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 2]
```diff
@@ -136,13 +140,17 @@ With this release, you can get the ID of a {microshift-short} cluster. When open
...
-//[id="microshift-4-16-networking-bug-fixes"]
-//==== Networking
+[discrete]
+[id="microshift-4-16-networking-bug-fixes"]
+=== Networking
 
-//[discrete]
-//[id="microshift-4-16-support-bug-fixes"]
-//==== Support
+Previously, the {microshift-short} load balancer controller tried to update the IP addresses of every `LoadBalancer` service in the cluster. Some of these services, such as those with a defined `loadBalancerClass`, have their own update procedures for external IPs. This conflicted with the {microshift-short} controller. Now, services that have a `loadBalancerClass` are filtered and IP addresses owned by other load balancer services are ignored by {microshift-short}. (link:https://issues.redhat.com/browse/OCPBUGS-30833[OCPBUGS-30833])
```
Comment (pacevedom) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:147]: this is lgtm

**Review by pacevedom — APPROVED** [review]
/lgtm

File: microshift_release_notes/microshift-4-16-release-notes.adoc [hunk 3]
```diff
@@ -136,13 +140,17 @@ With this release, you can get the ID of a {microshift-short} cluster. When open
...
+
+[discrete]
+[id="microshift-4-16-support-bug-fixes"]
+=== Support
+
+Previously, when `microshift-etcd` unexpectedly exited, {microshift-short} tried to restart so that `microshift-etcd` could restart, but there was a lingering unit fragment. Every attempt to restart `microshift-etcd` failed, making the system unusable. The `--collect` flag was added to the `systemd-run` invocation used to start `microshift-etcd`. The addition results in systemd cleaning up the unit fragment even if the unit failed. The system now recovers and restarts. (link:https://issues.redhat.com/browse/OCPBUGS-33588[OCPBUGS-33588])
```
Comment (aspauldi) [inline microshift_release_notes/microshift-4-16-release-notes.adoc:153]: Would you want to change "The" to "This"? I was confused by what addition was being referred to here. Just a suggestion though!

I think that systemd should be in the possessive format, as the text refers to "systemd**'s** cleaning up . . . "

**Review by aspauldi — COMMENTED** [review]
Hi Shauna, I noted only a couple of things. Looks good!

File: microshift_release_notes/microshift-4-16-release-notes.adoc (hunk 3 above)
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-16-release-notes.adoc:153]: https://spaces.redhat.com/display/CCSENABLE/Possessives+and+apostrophes (no possesives)

Comment (aspauldi) [reply microshift_release_notes/microshift-4-16-release-notes.adoc:153]: Thanks for the link. The info doesn't specifically call out gerunds, but that's okay!

**Pull request merged**